Sources/VLCTabBarCoordinator.swift

```diff
@@ -124,5 +124,24 @@ class VLCTabbarCooordinator: NSObject, VLCVideoControllerDelegate {
     }
 
     func videoViewControllerDidSelectSort(VLCVideoViewController: VLCVideoViewController) {
+        showSortOptions()
+    }
+
+    func showSortOptions() {
+        //should probably be in a coordinator as well
+        let sortOptionsAlertController = UIAlertController(title: NSLocalizedString("Sort by",comment: ""), message: nil, preferredStyle: .actionSheet)
+        let sortByNameAction = UIAlertAction(title: SortOption.alphabetically.string, style: .default) { action in
+        }
+        let sortBySizeAction = UIAlertAction(title: SortOption.size.string, style: .default) { action in
+        }
+        let sortbyDateAction = UIAlertAction(title: SortOption.insertonDate.string, style: .default) { action in
+        }
+        let cancelAction = UIAlertAction(title: NSLocalizedString("Cancel",comment:""), style: .cancel, handler: nil)
```

Comment:
Isn't there already a general translation identifier for the Cancel string in Alerts?

Reply:
You still need to set the title. pretty sure